test/integration/firebase.2.js

```diff
@@ -0,0 +1,27 @@
+const firebase = require('firebase/app')
```

Comment:
Since this is the same case as the `firebase.js` test case, we can just update that integration test with this source to avoid duplication.

Reply:
Done